mmv1/templates/terraform/examples/deploy_target.tf.erb

```diff
@@ -0,0 +1,23 @@
+resource "google_cloud_deploy_target" "<%= ctx[:primary_resource_id] %>" {
+  name          = "<%= ctx[:vars]['name'] %>"
+  description   = "Target Cluster"
+  annotations = {
+    generated-by = "magic-modules"
+  }
+  labels = {
+    env = "dev"
+  }
+  gke {
+    cluster = "${data.google_project.project.id}/locations/us-central1/clusters/dev"
```

Comment:
We can pull the project ID from an environment variable without using the project data source. You can search for `:PROJECT_ID` in `terraform.yaml` files to find examples of this

Reply:
Also, this cluster should probably be a reference to a real cluster we create during this test

Reply:
Will fix this

Reply:
You can reference a nonexistant cluster. The API doesn't validate it.